src/openrct2/network/network.cpp

```diff
@@ -1136,7 +1142,12 @@ void Network::Server_Send_TICK()
     // but debug version can check more often.
     static sint32 checksum_counter = 0;
     checksum_counter++;
+#ifdef DEBUG_DESYNC 
+    // Check every tick in debug version.
+    if (checksum_counter >= 1) {
+#else
```

Comment:
This is pretty horrible can't you just have different values defined for the different compile modes